tests/python/pants_test/pantsd/test_pantsd_integration.py

```diff
@@ -399,6 +399,22 @@ def test_pantsd_memory_usage(self):
                 ),
             )
 
+    def test_pantsd_max_memory_usage(self):
```

Comment:
Will probably want to sanity check that `test_pantsd_memory_usage` is still relevant. I know they do different things, but possible bit rot not. NB that we skip it due to flakiness.

Reply:
It is still relevant (see the comments on the ticket linked in the description of this PR), but I have not been able to follow up on re-enabling it.